test/Tools/NodeDocumentationMarkdownGeneratorTests/MarkdownGeneratorCommandTests.cs

```diff
@@ -317,7 +317,6 @@ public void ProducesCorrectOutputFromPackage()
                 "Examples.PeriodicUpdateExample.PointField.md",
                 "Examples.TransformableExample.ByGeometry.md",
                 "Examples.TransformableExample.Geometry.md",
-                "Examples.TransformableExample.TransformableExample.md",
```

Comment:
No documentation is now generated as this is a ctor with the invisible attribute, which takes effect.